query_table_create.go

```diff
@@ -212,6 +201,10 @@ func (q *CreateTableQuery) AppendQuery(fmter schema.Formatter, b []byte) (_ []by
 
 	b = q.appendPKConstraint(b, q.table.PKs)
 	b = q.appendUniqueConstraints(fmter, b)
+
+	if q.autoFKs {
+		q.createRelationFKs()
```

Comment:
As is, the query will be broken if it is appended multiple times.

Instead, you should probably rewrite the `createRelationFKs` so you generate the query by appending to `b`.

Reply:
Yep, you're totally right.  
Thanks for catching that, just pushed a test and a fix for it 🔧